Automattic/wp-calypso
# State: Move followers to redux

Moves followers to the redux state. See #5046 

This PR creates a branch in the redux tree for followers. It does NOT yet make use of this data in any UI components. A separate PR will handle a `queryFollowers` component to replace the current Flux implementation.

To test:
- check out this branch and run `npm run test` and ensure all follower tests are passing
- protip: in `state/followers/test/reducer.js`, `state/followers/test/selectors.js`, and `state/followers/test/utils.js` change the top level `describe()` to `describe.only()` to run just the follower tests

**Check for regressions**
Because I made a small change to a WPCOM api method, ensure that removing people still works. Select a site, go to People, select a Team member or Follower, and ensure that you can remove them from the site.

cc: @ebinnion @lezama
